docs/configuration.rst

```diff
@@ -10,337 +10,314 @@ see specific integration documentation for more details.
 
 The following environment variables for the tracer are supported:
 
+Unified Service Tagging
+-----------------------
 
 .. ddtrace-configuration-options::
+
    DD_ENV:
      description: |
          Set an application's environment e.g. ``prod``, ``pre-prod``, ``staging``. Added in ``v0.36.0``. See `Unified Service Tagging`_ for more information.
 
    DD_SERVICE:
      default: (autodetected)
+     
      description: |
          Set the service name to be used for this application. A default is
          provided for these integrations: :ref:`bottle`, :ref:`flask`, :ref:`grpc`,
          :ref:`pyramid`, :ref:`tornado`, :ref:`celery`, :ref:`django` and
          :ref:`falcon`. Added in ``v0.36.0``. See `Unified Service Tagging`_ for more information.
 
-   DD_SERVICE_MAPPING:
-     description: |
-         Define service name mappings to allow renaming services in traces, e.g. ``postgres:postgresql,defaultdb:postgresql``.
-
-   DD_TAGS:
-     description: |
-         Set global tags to be attached to every span. Value must be either comma or space separated. e.g. ``key1:value1,key2:value2`` or ``key1:value key2:value2``.
-
-         If a tag value is not supplied the value will be an empty string. e.g. ``key1,key2`` or ``key1 key2``.
-     version_added:
-       v0.38.0: Comma separated support added
-       v0.48.0: Space separated support added
-
-   DD_TRACE_PROPAGATION_HTTP_BAGGAGE_ENABLED:
-     type: Boolean
-     default: False
-     description: |
-         Enables propagation of baggage items through http headers with prefix ``ot-baggage-``.
-     version_added:
-       v2.4.0:
-
    DD_VERSION:
      description: |
          Set an application's version in traces and logs e.g. ``1.2.3``,
          ``6c44da20``, ``2020.02.13``. Generally set along with ``DD_SERVICE``.
 
          See `Unified Service Tagging`_ for more information.
+     
      version_added:
        v0.36.0:
 
-   DD_SITE:
-     default: datadoghq.com
-     description: |
-         Specify which site to use for uploading profiles and logs. Set to
-         ``datadoghq.eu`` to use EU site.
+Traces
+------
 
-   DD_TRACE_ENABLED:
-     type: Boolean
+.. ddtrace-configuration-options::
+   DD_<INTEGRATION>_DISTRIBUTED_TRACING:
      default: True
+     
      description: |
-         Enable sending of spans to the Agent. Note that instrumentation will still be installed and spans will be
-         generated.
+         Enables distributed tracing for the specified <INTEGRATION>.
+
      version_added:
-       v0.41.0: |
-           Formerly named ``DATADOG_TRACE_ENABLED``
+       v2.7.0:
```

Comment:
@mabdinur The version added here is different based on the integration. I'm thinking of changing this to just:

```suggestion
       Varies per integration.
```

Or should we list out the individual integrations here? 😅